guides/get-started-microservices-on-kubernetes/_includes/_constants.adoc

```diff
@@ -2,13 +2,15 @@
 :jaxrs-example-project-groupId: org.wildfly.examples
 :jaxrs-example-project-artifactId: jaxrs
 :jaxrs-example-project-version: 11.0.0.Final-SNAPSHOT
-:version-wildfly: 35.0.0.Final
-:version-wildfly-galleon-pack: 35.0.0.Final
+:version-wildfly: 35.0.1.Final
+:version-wildfly-galleon-pack: 35.0.1.Final
 :jakarta-jakartaee-api-version: 10.0.0
 :version-junit-jupiter-api: 5.11.3
-:version-wildfly-cloud-galleon-pack: 7.0.2.Final
-:version-wildfly-maven-plugin: 5.1.1.Final
+:version-wildfly-cloud-galleon-pack: 8.0.0.Final
+:version.wildfly.ai.feature.pack: 1.0.0-SNAPSHOT
```

Comment:
1.0.0-SNAPSHOT?

Reply:
`1.0.0-SNAPSHOT` is the version I get if I run:

```shell
git clone git@github.com:wildfly-extras/wildfly-ai-feature-pack.git
cd wildfly-ai-feature-pack
mvn clean install -DskipTests -Denforcer.skip
...
[INFO] Installing /tmp/wildfly-ai-feature-pack/ai-feature-pack/pom.xml to /home/tborgato/.m2/repository/org/wildfly/wildfly-ai-feature-pack/1.0.0-SNAPSHOT/wildfly-ai-feature-pack-1.0.0-SNAPSHOT.pom
...
```

Reply:
There is a release of it: https://github.com/wildfly-extras/wildfly-ai-feature-pack/releases/tag/0.4.2